### DISCUSSION TITLE
Transactions are not working on mainnet but working on testnet

### DISCUSSION THREAD

**Discussion by tahirahmadin** [Web3Auth/Web3Auth#1109] [Help]
Hey team,
I'm integrating the Social Login using web3Auth and able to implement successfully over the testnet environment and working flawlessly.
But once I move to mainnet transactions are not going through and giving "JSON RPC Error".
Is this because my project inside the dashboard is grey (Inactive) or any other issue?

<img width="387" alt="Screenshot at Dec 19 12-59-19" src="https://user-images.githubusercontent.com/25260610/208660182-06fd0539-24e9-4443-bfa3-88ffe4edadcc.png">

**Comment by shahbaz17** [comment]
Hey @tahirahmadin 

Could you share your `rpcTarget` here? It looks like it's down. Also, please share the initialization and login code snippet.

**Comment by tahirahmadin** [comment]
Started working now, Thanks!
